use-api.md

```diff
@@ -0,0 +1,144 @@
+|/objects/{id}|Returns data about an individual object.|
+|/objects/{id}/ancestors|Returns the ancestors of an object.|
+|/terms|Returns a list of terms. Terms are controlled values describing topics, geographic places or record formats.|
+|/terms/{id}|Returns data about an individual term.|
+|/search|Performs search queries across agents, collections, objects and terms.|
+|/search/{id}|Performs search queries across a specific agent, collection, object or term.
+|/schema/|Returns the OpenAPI schema for the RAC API.|
+
+### Parameters
+Use our [browseable API](https://api.rockarch.org) to see which parameters are available for which endpoints. For example, [https://api.rockarch.org/collections](https://api.rockarch.org/collections) lists the filter and sort fields, or parameters, that are available for that endpoint at the top of the webpage.
+
+| Parameter | Description | Example |
+|------|------|------|------|
+|limit|Number of results to return per page.|limit=50|
+|offset|Number of results to return per page.|offset=50|
+|id|Unique identifier. The id is used in an endpoint path to point to a specific collection, object, agents, etc.|/collections/2HnhFZfibK6SVVu86skz3k|
+|title|Filter or sort results by title.|title=world health organization|
+|start_date|Filter results by start date.|start_date=1932|
+|end_date|Filter results by start date.|end_date=1975|
+|category|Filter results by category term, including `person`, `collection`, or `organization`|category=person|
+|subject|Filter results by subject/topic term|subject=public health|
+|creator|Filter results by creator. Creators are the people, organizations, and families responsible for creating the records.|creator=adams, lillian|
+|online|Filter results by objects that are digital and available to view online.|online=true|
+|genre|Filters results by genre/format term, including `documents`, `photographs`, `moving image`, and `audio`.|genre=moving image|
+|query|Query for full-text search|query=yellow fever|
+|sort|Sort results by title, start_date, end_date, or type. By default the named property will be sorted ascending. Descending order can be achieved by appending an en dash (`-`) to the start of the property.|sort=title|
+
+## Example queries
+1. Use the `/search` endpoint to return the number of search matches for the query term "agriculture" that are in collections and have been categorized as photographs with dates between 1940 and 1950:
+
+```
+https://api.rockarch.org/search?&query=agriculture&category=collection&genre=photographs&start_date__gte=1940&end_date__lte=1950
+```
+
+**Note**: Appending `__gte` and `__lte` to the date parameters function as `greater than or equal to` and `less than or equal to`, allowing us to include any start and end dates in this decade instead of limiting ourselves to specific start and end dates. Similarly, `gt`= greater than and `ls`= less than.
```

Comment:
```suggestion
**Note**: Appending `__gte` and `__lte` to the date parameters function as `greater than or equal to` and `less than or equal to`, allowing us to include any start and end dates in this decade instead of limiting ourselves to specific start and end dates. Similarly, `gt`= greater than and `lt`= less than.
```